src/main/java/ch/njol/skript/expressions/ExprCreeperMaxFuseTicks.java

```diff
@@ -32,71 +33,63 @@
 import ch.njol.skript.expressions.base.SimplePropertyExpression;
 import ch.njol.util.coll.CollectionUtils;
 
-@Name("Creeper Max Fuse Ticks")
-@Description("The max fuse ticks that a creeper has.")
-@Examples("set target entity's max fuse ticks to 20 #1 second")
+@Name("Creeper Max Fuse Duration")
+@Description("The maximum fuse duration that a creeper has.")
+@Examples("set target entity's max fuse time to 1 second")
 @Since("2.5")
-public class ExprCreeperMaxFuseTicks extends SimplePropertyExpression<LivingEntity, Long> {
+public class ExprCreeperMaxFuseTicks extends SimplePropertyExpression<LivingEntity, Timespan> {
 	
 	static {
-		if(Skript.methodExists(LivingEntity.class, "getMaxFuseTicks"))
-			register(ExprCreeperMaxFuseTicks.class, Long.class, "[creeper] max[imum] fuse tick[s]", "livingentities");
+		if (Skript.methodExists(Creeper.class, "getMaxFuseTicks"))
+			register(ExprCreeperMaxFuseTicks.class, Timespan.class, "[creeper] max[imum] fuse (time|duration)", "livingentities");
```

Comment:
These expressions are used often, could this support the old pattern? Maybe add a deprecation warning for it so people notice

Reply:
I fear it wasn't used at all:
```
import:
  ch.njol.skript.Skript
  org.bukkit.entity.LivingEntity
  org.bukkit.entity.Creeper

command /test:
  trigger:
    send Skript.methodExists(LivingEntity.class, "getMaxFuseTicks")
    send Skript.methodExists(Creeper.class, "getMaxFuseTicks")
```
![image](https://user-images.githubusercontent.com/25848425/219717604-cda1bc86-b176-4a5a-a3e8-e33a1a728741.png)

Also unfindable in the docs...

Reply:
Sorry, I meant the pattern. What do you think of adding the `ticks` part back to the pattern?

Reply:
This entire expression never registered so neither did the pattern, no one has ever used this--at least I can't get it to work and can't find it in any documentation. Feel free to try the old pattern and see if it works for you.